### DISCUSSION TITLE
Usando GitHub tendo um windows e Mac

### DISCUSSION THREAD

**Discussion by Mysia14** [frontendbr/forum#2242] [Perguntas e Respostas]
Ola, 
Comecei usando o GitHub no meu computador pessoal, um Mac. Agora uso um windows para trabalho, mas usando a minha mesma conta do GitHub.
So que todo commit que faco no laptop windows nao aparece no diagrama da pagina principal. Mas aparece como se fosse uma contreuibuicao de outro usuario. Que criei na tentativa de adiconar uma chave para o windows.
Em resumo: como fazer que o Git hub reconheca tb o windows como o usuario principal e nao como contribuiter.
Muito obrigada.

**Comment by igorlvicente** [comment]
Pode mandar imagens do que você quer dizer?
Pelo que disseste, eu desconfio que seja a configuração de usuário que deves fazer:
```bash
git config --global user.name "YOUR NAME"
git config --global user.email "YOUR@EMAIL.com"
```
mas não tenho certeza se é isso que queres.

**Reply by Mysia14** [reply]
**Primeiramente muito obrigada @igorlvicente  por sua disposicao em ajudar.**

------------------------------------------------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------------------------------------------------
**Imagem 1**
![Screenshot 2022-02-10 at 11-22-52 Mysia14 MyLink-react-app One more task using React JS](https://user-images.githubusercontent.com/80990809/153472395-6eb2339d-a7d5-4e4b-bbce-4a46290e27e6.png)



---------------------------------------------------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------------------------------------------------
**Imagem 2**
![Screenshot 2022-02-10 at 11-23-35 Mysia14 cpnt262-a3](https://user-images.githubusercontent.com/80990809/153472411-37ebe314-14a6-4a6a-aa2e-3d10ea7f3020.png)


-------------------------------------------------------------------------------------------------------------------------------------------


**A 1a imagem é meu repositorio usando o windows depois que criei um a chave.**
A segunda imagem é como meus commits apareciam quando usava o mac que foi onde registrei minha 1a chave. 
Ai qaundo vou ver minha pagina principal onde mostra todas as contribuicoes e commits, so mostra os commits que fiz usando o mac ou somente os novos repos feito com windows.
Ou seja, no dia de ontem fiz varios commits, mas só mostra a contribuicao (menos verde).


---------------------------------------------------------------------------------------------------------------------------------------------

![Screenshot 2022-02-10 at 11-30-01 Mysia14 - Overview](https://user-images.githubusercontent.com/80990809/153473458-5c52738a-093e-45db-bbdf-37d86562e69a.png)


É como se meu github reconhecesse o que faco pelo windows como um colaborador e nao como principal usuario.

**Reply by ramonmetzker** [reply]
@Mysia14 Se o email e o nome que você colocou nas suas git configs forem diferentes das credenciais do seu github, ele não vai reconhecer que o commit foi seu. 
Se no seu Github seu email é 'joemail@mail.com' e o seu usuário é 'Joe User' você precisa setar suas configs com 
`git config user.name "Joe User"` e `git config user.email "joemail@mail.com"`
Isso vale para GitLab, Bitbucket, etc

**Reply by Mysia14** [reply]
Muito obrigada @ramonmetzker  vou fazer isso.

**Comment by theandersonn** [comment]
Você configuriu a chave ssh da sua máquina no github?

Se não:

1.
```bash
ssh-keygen
```

Copie o conteúdo gerado em `.ssh/id_rsa.pub`

2 Insira a chave SSH no GitHub
```bash
// Informe um nome para identificar a sua máquina no Github

// No Github vá em "Settings > SSH and GPG Keys" > Clique em New SSH Key
```

**Reply by Mysia14** [reply]
Muito obrigada @theandersonn . Eu fiz como vc sugeriu mas diz que  achave ja existe e nao me deu a opcao de overwrite. : (

**Reply by theandersonn** [reply]
Entendi, é necessário fazer esse procedimento em cada computador que vc utiliza e no github add entradas diferentes.
